clang/lib/Driver/ToolChains/CommonArgs.cpp

```diff
@@ -1252,6 +1252,9 @@ void tools::addArchSpecificRPath(const ToolChain &TC, const ArgList &Args,
                     options::OPT_fno_rtlib_add_rpath, false))
     return;
 
+  if (TC.getTriple().isOSAIX()) // AIX doesn't support -rpath option.
```

Comment:
nit: We did add `-rpath` support to the clang driver, essentially we fold all the `rpath` options into a `blibpath` for the library. 

That said, I'm not convinced we really want to add anything here anyway right now, so maybe a `TODO` would be appropriate.
```suggestion
  if (TC.getTriple().isOSAIX()) // TODO: AIX doesn't support -rpath option.
```

Reply:
Right. Will change to the suggestion.